dat/spob/burnan.xml

```diff
@@ -26,7 +26,8 @@
    <shipyard/>
   </services>
   <commodities/>
-  <description>Burnan counts as the unofficial local capital. Aesir is far away and many government services are better rendered locally than over a distance of many hyperspace jumps. Burnan has a large political district in its primary city, this also houses one of the largest Sirian temples in all of Sirius space. Due to the highly political nature of the planet, the cities and districts are designed with specific echelons in mind and mixing is kept to a minimum.</description>
+  <description>As Aesir is so far away and many government services  are better rendered locally than over a distance of many hyperspace jumps, Burnan serves as the local capital in this sector of Sirius space. Because of this, the primary city of Burnan sports an atypically large political district in its primary city, as well as one of the largest Sirian temples in all of Sirius space.
```

Comment:
```suggestion
  <description>As Aesir is so far away and many government services are better rendered locally than over a distance of many hyperspace jumps, Burnan serves as the local capital in this sector of Sirius space. Because of this, the primary city of Burnan sports an atypically large political district in its primary city, as well as one of the largest Sirian temples in all of Sirius space.
```